runtime/src/bank/partitioned_epoch_rewards/mod.rs

```diff
@@ -276,9 +271,8 @@ mod tests {
         }
 
         /// Return the total number of blocks in reward interval (including both calculation and crediting).
-        pub(in crate::bank) fn get_reward_total_num_blocks(&self, rewards: &StakeRewards) -> u64 {
-            self.get_reward_calculation_num_blocks()
-                + self.get_reward_distribution_num_blocks(rewards)
+        fn get_reward_total_num_blocks(&self, rewards: &StakeRewards) -> u64 {
+            REWARD_CALCULATION_NUM_BLOCKS + self.get_reward_distribution_num_blocks(rewards)
```

Comment:
Looks like this can be cleaned up since all the uses of this method don't seem necessary in the tests below